pkg/ssh/ssh.go

```diff
@@ -376,7 +386,9 @@ func (l *SSHTunnelList) removeAndReAdd(e sshTunnelEntry) {
 	go l.createAndAddTunnel(e.Address)
 }
 
-func (l *SSHTunnelList) Dial(ctx context.Context, net, addr string) (net.Conn, error) {
+// Dial returns a new Connection interface for the input tunnel. If the tunnel is not
+// open, an error is returned.
```

Comment:
nit: The function also connects to the corresponding addr through a tunnel in the TunnelList.

Reply:
Thanks Jeff! Fixed in the latest commit.